src/app/commands/toggle_other_layers_visibility.cpp

```diff
@@ -0,0 +1,133 @@
+// Aseprite
+// Copyright (C) 2025  Igara Studio S.A.
+//
+// This program is distributed under the terms of
+// the End-User License Agreement for Aseprite.
+
+#include "app/commands/command.h"
+#include "app/commands/new_params.h"
+#include "app/context_access.h"
+#include "app/doc.h"
+#include "app/i18n/strings.h"
+#include "doc/layer.h"
+
+namespace app {
+
+struct ToggleOtherLayersVisibilityParams : public NewParams {
+  Param<int> layerId{ this, 0, "layerId" };
+};
+
+class ToggleOtherLayersVisibilityCommand
+  : public CommandWithNewParams<ToggleOtherLayersVisibilityParams> {
+public:
+  ToggleOtherLayersVisibilityCommand();
+
+private:
+  void onExecute(Context* ctx) override;
+  std::string onGetFriendlyName() const override;
+};
+
+ToggleOtherLayersVisibilityCommand::ToggleOtherLayersVisibilityCommand()
+  : CommandWithNewParams<ToggleOtherLayersVisibilityParams>(
+      CommandId::ToggleOtherLayersVisibility(),
+      CmdUIOnlyFlag)
+{
+}
+
+void showChildrenLayers(const LayerGroup* layerGroup, Doc* doc)
```

Comment:
It looks like a static standalone function used only in this file:
```suggestion
static void show_children_layers(const LayerGroup* layerGroup, Doc* doc)
```